build_resources/linux/apparmor-profile

```diff
@@ -0,0 +1,9 @@
+abi <abi/4.0>,
+include <tunables/global>
+
+profile hathor-wallet /opt/Hathor\ Wallet/hathor-wallet flags=(default_allow) {
```

Comment:
Changed to default_allow, it is the must correct flag value. For more details refer to: https://manpages.ubuntu.com/manpages/noble/man5/apparmor.d.5.html